<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resource_widget</class>
 <widget class="QWidget" name="resource_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_main">
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="lbl_image">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>N/A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_spacer">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="layout_res_info">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_name">
        <property name="text">
         <string>Name :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_name">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_width">
        <property name="text">
         <string>Width :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_width_value">
        <property name="text">
         <string>0 px</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_height">
        <property name="text">
         <string>Height :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_height_value">
        <property name="text">
         <string>0 px</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
